mobile/src/libs/components/icon/libs/maps/icon-name-to-icon/icon-name-to-icon.map.ts

```diff
@@ -1,40 +1,40 @@
-import backArrow from '#assets/img/icons/back-arrow.svg';
-import backward from '#assets/img/icons/backward.svg';
-import bell from '#assets/img/icons/bell.svg';
-import chat from '#assets/img/icons/chat.svg';
-import close from '#assets/img/icons/close.svg';
-import download from '#assets/img/icons/download.svg';
-import forward from '#assets/img/icons/forward.svg';
-import home from '#assets/img/icons/home.svg';
-import next from '#assets/img/icons/next.svg';
-import pause from '#assets/img/icons/pause.svg';
-import play from '#assets/img/icons/play.svg';
-import plus from '#assets/img/icons/plus.svg';
-import previous from '#assets/img/icons/previous.svg';
-import signOut from '#assets/img/icons/sign-out.svg';
-import upload from '#assets/img/icons/upload.svg';
-import user from '#assets/img/icons/user.svg';
+import BackArrow from '#assets/img/icons/back-arrow.svg';
+import Backward from '#assets/img/icons/backward.svg';
+import Bell from '#assets/img/icons/bell.svg';
+import Chat from '#assets/img/icons/chat.svg';
+import Close from '#assets/img/icons/close.svg';
+import Download from '#assets/img/icons/download.svg';
+import Forward from '#assets/img/icons/forward.svg';
+import Home from '#assets/img/icons/home.svg';
+import Next from '#assets/img/icons/next.svg';
+import Pause from '#assets/img/icons/pause.svg';
+import Play from '#assets/img/icons/play.svg';
+import Plus from '#assets/img/icons/plus.svg';
+import Previous from '#assets/img/icons/previous.svg';
+import SignOut from '#assets/img/icons/sign-out.svg';
+import Upload from '#assets/img/icons/upload.svg';
+import User from '#assets/img/icons/user.svg';
```

Comment:
Why did you change this? Liza said it should be in lowercase

Reply:
@danilittaa  because i have delete icon, and writing just 'delete' is forbidden in code

Reply:
rename it to trashBox